beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -275,41 +276,25 @@ func (vs *Server) ProposeBeaconBlock(ctx context.Context, req *ethpb.GenericSign
 
 // PrepareBeaconProposer caches and updates the fee recipient for the given proposer.
 func (vs *Server) PrepareBeaconProposer(
```

Comment:
We have to make sure validator client still pushes these calls even fee recipient is only defined on the beacon node

Reply:
Nice catch, I'll have the beacon node set it at launch once. This way the vc will overwrite it if it's also defined and preserves the current behavior